### PR TITLE
Add typedoc comments for layer 2 classes

### PR DESCRIPTION
**Pull request by platfowner**
Change summary:
- Add typedoc comments for layer 2 classes

Todos:
- Add typedoc comments for event handler modules

Related issues:
- https://github.com/ainblockchain/ain-blockchain/issues/1207

### CODE REVIEW

File: src/ain-db/db.ts [hunk 1]
```diff
@@ -19,8 +23,8 @@ export default class Database {
 
   /**
    * Returns a reference instance of the given path.
-   * @param {String} path
-   * @return {Reference} A reference instance of the given path.
+   * @param {String} path The path to refer to.
+   * @returns {Reference} A reference instance of the given path.
```
Comment (jiyoung-an) [inline src/ain-db/db.ts:27]: What is difference between @return and @returns?

**Review by jiyoung-an — APPROVED** [review]
LGTM with some questions, Thanks!

File: src/ain-db/db.ts (hunk 1 above)
Comment (platfowner) [reply src/ain-db/db.ts:27]: It seems @returns is the standard: https://typedoc.org/tags/returns/

File: src/he/desilo.js [hunk 2]
```diff
@@ -5,9 +5,13 @@
 const SEAL = require('node-seal');
 
 /**
- * wrapper Error class for DesiloSeal
+ * A wrapper Error class for DesiloSeal.
  */
 class DesiloSealError extends Error {
```
[the remark this answers is not in the document]
Comment (platfowner) [reply src/he/desilo.js:10]: It a HE solution from Desilo Inc.

**Review by woomurf — APPROVED** [review]
LGTM, Thanks!

**Comment by platfowner** [comment]
Thanks for the review! @jiyoung-an @woomurf  Now merging..

**Pull request merged**